Add assert_polys_grouped_by function

nkorinek opened the pull request:
Added a function to test that polygons were grouped by a certain attribute, much like the functions we have that do this for lines and points. Also added some tests. This pr addressed issue #225. I can add this function to our vector vignette once this https://github.com/earthlab/matplotcheck/pull/220 pr is merged.

nkorinek committed:
Merge pull request #1 from earthlab/master

Updating files

pyup-bot committed:
# This is a combination of 2 commits.
# This is the 1st commit message:

Update setuptools from 42.0.2 to 45.2.0 (#190)

raster inherits from vector rather than base (#76)

Allow geodataframes in assert_polygons (#188)

fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

update changelog for release

Bump version: 0.1.1 → 0.1.2

Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Update codecov to 2.0.16 (#202)

* Update codecov from 2.0.15 to 2.0.16

* update from master (#211)

* Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>
Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

Update setuptools from 45.2.0 to 46.0.0 (#215)

Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

Update pytest from 5.3.5 to 5.4.1

merge

Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Add tests to the Autograde module (#224)

* Added tests for the autograde module

* removed unneccesary import

* Fixing issues that were failing in base_data

* codacy

* flake8

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Updated changelog

* flake8 fix

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Hist bin midpoints (#204)

* Add a get and assert function for bin midpoints, as well as tests for those functions

* Updated docstrings

* Added changelog changes and fixed minor formatting issue

* small flake8 fix

* Added midpoints functionatlity to the vignette for histogram testing

* Adding in changes suggested on GitHub1

* Fixed function description

* Took out all instances of  in example

* update docs

* Fixed small bug with section titles

* Update matplotcheck/tests/test_base_data.py

* reworded changelog

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Update setuptools from 46.1.1 to 46.1.3 (#231)

Title assert accepts strings (#229)

* Fixed assert_title_contains to take strings as well as lists

* Better implementation

* Changelog update

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

M2r remove (#247)

* Removed m2r from matplotcheck, and reformated code of conduct

* Seeing if rebase works

* removed an m2r import

* Fixing up the docs a bit

* changelog updates

Allow geodataframes in assert_polygons (#188)

fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Get images function (#193)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Bump version: 0.1.1 → 0.1.2

Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

# This is the commit message #2:

Update setuptools from 46.1.1 to 46.1.3 (#231)

pyup-bot committed:
Update setuptools from 42.0.2 to 45.2.0 (#190)

nkorinek committed:
Merge pull request #2 from earthlab/master

Update setuptools from 42.0.2 to 45.2.0 (#190)

Commit:
raster inherits from vector rather than base (#76)

nkorinek committed:
Merge pull request #3 from earthlab/master

raster inherits from vector rather than base (#76)

nkorinek committed:
Allow geodataframes in assert_polygons (#188)

nkorinek committed:
Merge pull request #4 from earthlab/master

Allow geodataframes in assert_polygons (#188)

Commit:
fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

nkorinek committed:
Merge pull request #5 from earthlab/master

Updating from master

Commit:
update changelog for release

Commit:
Bump version: 0.1.1 → 0.1.2

nkorinek committed:
Merge pull request #7 from earthlab/master

updating

nkorinek committed:
Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

nkorinek committed:
Merge pull request #8 from earthlab/master

Add flake8 fix [WIP] (#199)

nkorinek committed:
Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

nkorinek committed:
Merge pull request #9 from earthlab/master

Assert string accept spaces (#205)

pyup-bot committed:
Update codecov to 2.0.16 (#202)

* Update codecov from 2.0.15 to 2.0.16

* update from master (#211)

* Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>
Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

nkorinek committed:
Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

pyup-bot committed:
Update setuptools from 45.2.0 to 46.0.0 (#215)

nkorinek committed:
Merge pull request #10 from earthlab/master

update

pyup-bot committed:
Update pytest from 5.3.5 to 5.4.1

nkorinek committed:
Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

nkorinek committed:
Merge pull request #11 from earthlab/master

Add Assert points function & cleanup duplicate methods (#203)

nkorinek committed:
Merge pull request #12 from earthlab/master

Update pytest from 5.3.5 to 5.4.1

nkorinek committed:
Merge pull request #13 from earthlab/master

update

nkorinek committed:
Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

nkorinek committed:
Merge pull request #14 from earthlab/master

update

nkorinek committed:
Add tests to the Autograde module (#224)

* Added tests for the autograde module

* removed unneccesary import

* Fixing issues that were failing in base_data

* codacy

* flake8

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Updated changelog

* flake8 fix

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

nkorinek committed:
Merge pull request #15 from earthlab/master

Add tests to the Autograde module (#224)

nkorinek committed:
Hist bin midpoints (#204)

* Add a get and assert function for bin midpoints, as well as tests for those functions

* Updated docstrings

* Added changelog changes and fixed minor formatting issue

* small flake8 fix

* Added midpoints functionatlity to the vignette for histogram testing

* Adding in changes suggested on GitHub1

* Fixed function description

* Took out all instances of  in example

* update docs

* Fixed small bug with section titles

* Update matplotcheck/tests/test_base_data.py

* reworded changelog

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

pyup-bot committed:
Update setuptools from 46.1.1 to 46.1.3 (#231)

nkorinek committed:
Merge pull request #16 from earthlab/master

update

nkorinek committed:
Added in assert_polys_grouped_by_type along with tests for that function

nkorinek commented:
@lwasser here for review whenever you get the chance!

nkorinek committed:
Title assert accepts strings (#229)

* Fixed assert_title_contains to take strings as well as lists

* Better implementation

* Changelog update

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Commit:
Merge branch 'master' into polygon_grouped_by

lwasser commented:
thank you @nkorinek what assignment were you testing this on? i just want to test it locally. maybe one of the vector assignments?

nkorinek committed:
Merge pull request #17 from earthlab/master

Title assert accepts strings (#229)

nkorinek commented:
@lwasser ya I would test this on the clip assignment, towards the end, when there are the polygons with the values based on land/water area.

nkorinek committed:
M2r remove (#247)

* Removed m2r from matplotcheck, and reformated code of conduct

* Seeing if rebase works

* removed an m2r import

* Fixing up the docs a bit

* changelog updates

nkorinek committed:
Merge pull request #18 from earthlab/master

M2r remove (#247)

nkorinek committed:
merge

nkorinek committed:
merge

nkorinek committed:
Updating in order to squash commits!

Commit:
fix copyright on docs (#244)

Commit:
Add pillow (#254)

* add pillow as a dev requirement

* update for pillow addition

Commit:
Merge branch 'master' into polygon_grouped_by

nkorinek committed:
Merge

nkorinek committed:
Merge branch 'earthlab-master'

nkorinek committed:
Changelog to rst (#267)

* Update setuptools from 42.0.2 to 45.2.0 (#190)

* raster inherits from vector rather than base (#76)

* Allow geodataframes in assert_polygons (#188)

* fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

* update changelog for release

* Bump version: 0.1.1 → 0.1.2

* Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

* Update codecov to 2.0.16 (#202)

* Update codecov from 2.0.15 to 2.0.16

* update from master (#211)

* Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>
Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

* Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

* Update setuptools from 45.2.0 to 46.0.0 (#215)

* Update pytest from 5.3.5 to 5.4.1

* Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

* merge

* Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Add tests to the Autograde module (#224)

* Added tests for the autograde module

* removed unneccesary import

* Fixing issues that were failing in base_data

* codacy

* flake8

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Updated changelog

* flake8 fix

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Hist bin midpoints (#204)

* Add a get and assert function for bin midpoints, as well as tests for those functions

* Updated docstrings

* Added changelog changes and fixed minor formatting issue

* small flake8 fix

* Added midpoints functionatlity to the vignette for histogram testing

* Adding in changes suggested on GitHub1

* Fixed function description

* Took out all instances of  in example

* update docs

* Fixed small bug with section titles

* Update matplotcheck/tests/test_base_data.py

* reworded changelog

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Update setuptools from 46.1.1 to 46.1.3 (#231)

* Title assert accepts strings (#229)

* Fixed assert_title_contains to take strings as well as lists

* Better implementation

* Changelog update

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* M2r remove (#247)

* Removed m2r from matplotcheck, and reformated code of conduct

* Seeing if rebase works

* removed an m2r import

* Fixing up the docs a bit

* changelog updates

* # This is a combination of 2 commits.
# This is the 1st commit message:

Update setuptools from 42.0.2 to 45.2.0 (#190)

raster inherits from vector rather than base (#76)

Allow geodataframes in assert_polygons (#188)

fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

update changelog for release

Bump version: 0.1.1 → 0.1.2

Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Update codecov to 2.0.16 (#202)

* Update codecov from 2.0.15 to 2.0.16

* update from master (#211)

* Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

* Assert string accept spaces (#205)

* Made it so that assert_string_contains accepts key words with spaces

* black

* renaming function

* Showed assert_title_contains working with a space in the keyword in vignette

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>
Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

Update setuptools from 45.2.0 to 46.0.0 (#215)

Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

Update pytest from 5.3.5 to 5.4.1

merge

Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Add tests to the Autograde module (#224)

* Added tests for the autograde module

* removed unneccesary import

* Fixing issues that were failing in base_data

* codacy

* flake8

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Update matplotcheck/tests/test_autograde.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* Updated changelog

* flake8 fix

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Hist bin midpoints (#204)

* Add a get and assert function for bin midpoints, as well as tests for those functions

* Updated docstrings

* Added changelog changes and fixed minor formatting issue

* small flake8 fix

* Added midpoints functionatlity to the vignette for histogram testing

* Adding in changes suggested on GitHub1

* Fixed function description

* Took out all instances of  in example

* update docs

* Fixed small bug with section titles

* Update matplotcheck/tests/test_base_data.py

* reworded changelog

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Update setuptools from 46.1.1 to 46.1.3 (#231)

Title assert accepts strings (#229)

* Fixed assert_title_contains to take strings as well as lists

* Better implementation

* Changelog update

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

M2r remove (#247)

* Removed m2r from matplotcheck, and reformated code of conduct

* Seeing if rebase works

* removed an m2r import

* Fixing up the docs a bit

* changelog updates

Allow geodataframes in assert_polygons (#188)

fixing a few syntax errors (#194)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* fixing a few syntax errors

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* remove tabls

Co-authored-by: Nathan Korinek <nako1890@colorado.edu>

Get images function (#193)

* First draft of get_images function

* Added a test for get_image

* Changelog update

* minor formatting change.

* black changes

* Update matplotcheck/raster.py

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Bump version: 0.1.1 → 0.1.2

Add flake8 fix [WIP] (#199)

* add flake 8

* yay cleanup

* update change log

* add flake 8 dep

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* more line length issues

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* fixed test_base_axis.py

* fixed text_base_data.py

* fixed test_base_legends.py

* fixed test_base_titles_captions.py

* Format files to be flake8 acceptable (#197)

* autograde flake8 fixes

* Fixed base to be flake8 compliant (phew)

* Made folium.py flake8 compliant

* made notebook.py flake8 compliant

* Fixed raster.py for flake8

* Fixed timeseries.py

* fixed conftest.py

* fixed test_base.py

* Fixed test_raster.py and issue with legends tests

* black

* more black changes

* Fixing merge conflicts that were found

* Changing accepted changes

* minor changes

* Changing to make CI happy, modified to pass black and make -B docs

* Reformatting to make black and flake8 happy. Also updated crs assignment

* Fixed formatting issue with timeseries

* Better fix for timeseries formatting

* Black

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Contributors update (#213)

* Added contributor rst file and moved that information out of the README file

* Implemented requested changes to README

Add Assert points function & cleanup duplicate methods (#203)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* Taking out redundant tests

* Typo

* Fixing how vector checks for truth value of a dataframe

Additional vector tests (#212)

* Added a get_points() and assert_points() function to the vector tester.

* Added in get_points() and assert_points() functions with tests

* Added proper documentation

* Small fix to please codacy

* black

* Updated changelog

* First round of tests for vector

* black

* Rough draft for bug fix

* Fixed bug with multiple geometries plotted alongside bug with identical x values causing failure!

* typo

* Fixed small bug with markersize

* Added comments explaining code

* rough drafts of more tests

* Added more tests for legends

* Added more tests, and took out broken tests

* small codacy fix

* Fixed test!

* Taking out redundant tests

* Took out unneccesary tests

* Added plt.close

* Added more tests

* Typo

* more tests for uncovered parts of the vector file

* black

* Fixed issues with vector checking truth value of dataframe, and added tests

* Fixing how vector checks for truth value of a dataframe

* Added more coverage!

* Broke tests up into individual files

* black

* Added tests for outlier cases

* Update matplotcheck/tests/test_points.py

typo

Co-Authored-By: Leah Wasser <leah.wasser@colorado.edu>

* took plt.gca() out of tests

* Added changes suggested on GitHub

* Update CHANGELOG.md

* fix import order

* import order fix

* import order

* import order

Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

# This is the commit message #2:

Update setuptools from 46.1.1 to 46.1.3 (#231)

* fix copyright on docs (#244)

* Add pillow (#254)

* add pillow as a dev requirement

* update for pillow addition

* Changed the changelog to an rst file instead of a markdown file and added it to the docs build.

* Put changelog changes into the new changelog lol

Co-authored-by: pyup.io bot <github-bot@pyup.io>
Co-authored-by: Leah Wasser <leah.wasser@colorado.edu>

Commit:
setup greetings! (#257)

nkorinek committed:
Update from master

nkorinek committed:
Merge branch 'polygon_grouped_by' of https://github.com/nkorinek/matplotcheck into polygon_grouped_by

nkorinek committed:
Merge branch 'master' into polygon_grouped_by

nkorinek committed:
Fixed issue with changelog build

nkorinek committed:
Merge branch 'polygon_grouped_by' of https://github.com/nkorinek/matplotcheck into polygon_grouped_by
